digraph "unitGraph" {
    "$r0 = new compiler.c2.cr6646020.Tester"
    "specialinvoke $r0.<compiler.c2.cr6646020.Tester: void <init>()>()"
    "r1 = $r0"
    "specialinvoke r1.<compiler.c2.cr6646020.Tester: void test()>()"
    "goto [?= $r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()]"
    "$r5 := @caughtexception"
    "$r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()"
    "return $r2"
    "$r4 := @caughtexception"
    "return \"Error during result conversion to String\""
    "$r3 := @caughtexception"
    "return \"Error during test execution\""
    "$r0 = new compiler.c2.cr6646020.Tester"->"specialinvoke $r0.<compiler.c2.cr6646020.Tester: void <init>()>()";
    "$r0 = new compiler.c2.cr6646020.Tester"->"$r3 := @caughtexception";
    "specialinvoke $r0.<compiler.c2.cr6646020.Tester: void <init>()>()"->"r1 = $r0";
    "specialinvoke $r0.<compiler.c2.cr6646020.Tester: void <init>()>()"->"$r3 := @caughtexception";
    "r1 = $r0"->"$r5 := @caughtexception";
    "r1 = $r0"->"$r3 := @caughtexception";
    "r1 = $r0"->"specialinvoke r1.<compiler.c2.cr6646020.Tester: void test()>()";
    "specialinvoke r1.<compiler.c2.cr6646020.Tester: void test()>()"->"$r5 := @caughtexception";
    "specialinvoke r1.<compiler.c2.cr6646020.Tester: void test()>()"->"$r3 := @caughtexception";
    "specialinvoke r1.<compiler.c2.cr6646020.Tester: void test()>()"->"goto [?= $r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()]";
    "goto [?= $r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()]"->"$r4 := @caughtexception";
    "goto [?= $r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()]"->"$r3 := @caughtexception";
    "goto [?= $r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()]"->"$r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()";
    "$r5 := @caughtexception"->"$r4 := @caughtexception";
    "$r5 := @caughtexception"->"$r3 := @caughtexception";
    "$r5 := @caughtexception"->"$r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()";
    "$r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()"->"$r4 := @caughtexception";
    "$r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()"->"$r3 := @caughtexception";
    "$r2 = virtualinvoke r1.<compiler.c2.cr6646020.Tester: java.lang.String toString()>()"->"return $r2";
    "$r4 := @caughtexception"->"return \"Error during result conversion to String\"";
    "$r4 := @caughtexception"->"$r3 := @caughtexception";
    "$r3 := @caughtexception"->"return \"Error during test execution\"";
}
